src/component/movies/MovieLikes.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addLike, removeLike, flipLike } from '../../store/actions/MovieActions';
+import '../../styles/css/movies/movie_like.css'
+
+class MovieLikes extends Component {
+  
+    performLike = (like) => {
+        if (this.props.movie.user_liked_or_disliked === 0) {
+            this.props.addLike({movie: this.props.movie.id, like: like});
+        }
+        else if (this.props.movie.user_liked_or_disliked === like) {
+            this.props.removeLike({movie: this.props.movie.id, like: like});
+        } else {
+            this.props.flipLike({movie: this.props.movie.id, like: like});
```

Comment:
sta radi flip like ?

Reply:
napisao sam ti na BE sta treba uraditi povodom ovoga

Reply:
Izmenio to na BE, pa cu i ovde samo addLike bez tog flipLike